app/views/contacts/show.html.erb

```diff
@@ -2,53 +2,56 @@
 <p class="notice"><%= notice %></p>
 <% end %>
 
-<%= link_to contacts_path, class: "go-back" do %>
-  <%= image_tag "back.svg", alt: "" %><span>Go back</span>
-<% end %>
+<div class="tab_buttons margin-bottom-70">
+  <%= link_to 'Profile', contact_path(@contact), class: 'active' %>
+  <%= link_to "View #{@contact.first_name}'s Tasks (#{@contact.tasks.count})", controller: :contacts, action: :show_tasks %>
```

Comment:
Not sure the wording here makes sense - TBF - I couldn't see any wording on the designs.
Instead of "`FirstName`'s Tasks" which sounds like its the contact who needs to do something, how about:
`View tasks to do for first_name`
Maybe check with Nicholas/Jaye/Tim - not too important, but words matter :-)